packages/api/src/errors.js

```diff
@@ -77,3 +77,65 @@ export class InvalidCidError extends Error {
   }
 }
 InvalidCidError.CODE = 'ERROR_INVALID_CID'
+
+export class PinningServiceApiError extends Error {
+  /**
+   *
+   * @param {string} reason
```

Comment:
Can we follow Error namings here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error#parameters and use message instead of reason?